__tests__/utils/log.spec.js

```diff
@@ -47,19 +47,19 @@ describe('log functions', () => {
     it('outputs the correct string when not given a templateBanner', () => {
       goToStep(1);
       expect(console.log.mock.calls.join('\n')).toMatchInlineSnapshot(`
-        "[2J
+"[2J
 
-         _   _     _               _____                    _       _
-        | | | |___(_)_ __   __ _  |_   _|__ _ __ ___  _ __ | | __ _| |_ ___
-        | | | / __| | '_ \\\\ / _\` |   | |/ _ \\\\ '_ \` _ \\\\| '_ \\\\| |/ _\` | __/ _ \\\\
-        | |_| \\\\__ \\\\ | | | | (_| |   | |  __/ | | | | | |_) | | (_| | ||  __/
-         \\\\___/|___/_|_| |_|\\\\__, |   |_|\\\\___|_| |_| |_| .__/|_|\\\\__,_|\\\\__\\\\___|
-                           |___/                     |_|              vpackageVersionMock
+ _   _     _               _____                    _       _
+| | | |___(_)_ __   __ _  |_   _|__ _ __ ___  _ __ | | __ _| |_ ___
+| | | / __| | '_ \\ / _\` |   | |/ _ \\ '_ \` _ \\| '_ \\| |/ _\` | __/ _ \\
+| |_| \\__ \\ | | | | (_| |   | |  __/ | | | | | |_) | | (_| | ||  __/
+ \\___/|___/_|_| |_|\\__, |   |_|\\___|_| |_| |_| .__/|_|\\__,_|\\__\\___|
```

Comment:
Did this change the splash image at all?

Reply:
nope, it's rendering as expected. This changed after upgrading deps